ironfish/src/syncer.ts

```diff
@@ -33,11 +43,14 @@ export class Syncer {
   readonly speed: Meter
   readonly downloadSpeed: RollingAverage
 
-  state: 'stopped' | 'idle' | 'stopping' | 'syncing'
+  state: SyncerState
   stopping: Promise<void> | null
   eventLoopTimeout: SetTimeoutToken | null
   loader: Peer | null = null
   blocksPerMessage: number
+  nextMeasureTime = 0
+  numberOfMeasurements = 0
```

Comment:
could you call it simply `measurements`?

Reply:
`numberOfMeasurements` feels more clear at a glance in my opinion. Is `measurements` some object containing multiple types of measurements? Is it an array of a measurement object? I wouldn't expect it to be a counter of sorts if I'm just casually glancing at this file and seeing it for the first time

Reply:
makes sense, i'll resolve